packages/shared/lib/services/connection.service.ts

```diff
@@ -251,12 +251,6 @@ class ConnectionService {
         credentials: BasicApiCredentials | ApiKeyCredentials,
         logContextGetter: LogContextGetter
     ) {
-        const connection = await this.checkIfConnectionExists(connection_id, provider_config_key, environmentId);
-
-        if (connection) {
-            throw new NangoError('connection_already_exists');
-        }
-
```

Comment:
I wonder if we should remove all usage of the `checkIfConnectionExists` function. @khaliqgant do you see any issue with that?

Reply:
This is used quite a bit in the `connection.service` so I see no reason to remove that function